euler.rosinstall

```diff
@@ -7,3 +7,6 @@
     version: euler_integration}
 - git: {local-name: sicktoolbox_wrapper, uri: 'https://github.com/swri-robotics/sicktoolbox_wrapper.git',
     version: euler_integration}
+- git: {local-name: navigation, uri: 'https://github.com/DLu/navigation.git',
```

Comment:
That sounds like a good plan. The changes will eventually be rolled into the main navigation repo, once we create an indigo branch.